dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -596,6 +597,13 @@ def self.get_script(request)
     end
   end
 
+  # showing script levels by script id is no longer supported.
```

Comment:
Do you mind adding a comment about why we have to explicitly check rather than just not searching for a unit by id? It took me a couple minutes to realize that we don't want to change `Unit.get_from_cache` to not look up by ID.